fn_ipvoid/fn_ipvoid/lib/resilient_common.py

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+BASE_URL = 'https://endpoint.apivoid.com/{}/v1/pay-as-you-go/?key={}&{}'
+
+def get_url(api_key, query_type, value):
+    """
+    Method Will return appropriate URL based on query type    
+    """
+    type_api_pattern_mapping = {
+        'IP Reputation': ['iprep', 'ip={}'],
+        'Domain Blacklist': ['domainbl', 'host={}'],
+        'DNS Lookup': ['dnslookup', 'action=dns-a&host={}'],
+        'Email Verify': ['emailverify', 'host={}'],
+        'Threat Log': ['threatlog', 'host={}'],
+        'SSL Info': ['sslinfo', 'host={}']
+    }
+
+    url_pattern1 = type_api_pattern_mapping.get(query_type)[0]
```

Comment:
you should have some error checking if the type_api_pattern_mapping key isn't found

Reply:
Great suggestion, I'll put that